Contracts worker haven't registered get_transactions_per_safe_app_task

**Describe the bug**
`get_transactions_per_safe_app_task` is registered to be executed one time per week. https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/history/management/commands/setup_service.py#L121
But when the worker receive the task return the following:
```
[ERROR] [???/???] Received unregistered task of type 'safe_transaction_service.analytics.tasks.get_transactions_per_safe_app_task'.
The message has been ignored and discarded.
```

**Expected behavior**
Run the task without errors

**Additional context**
The task is added to the contract queue here https://github.com/safe-global/safe-transaction-service/blob/master/config/settings/base.py#L248

--- a/config/settings/base.py
+++ b/config/settings/base.py
@@ -93,6 +93,7 @@ THIRD_PARTY_APPS = [
     "rest_framework.authtoken",
 ]
 LOCAL_APPS = [
+    "safe_transaction_service.analytics.apps.AnalyticsConfig",
     "safe_transaction_service.contracts.apps.ContractsConfig",
     "safe_transaction_service.history.apps.HistoryConfig",
     "safe_transaction_service.notifications.apps.NotificationsConfig",

